Add python and ruby modules.

sebastianwelsh opened the pull request:
<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/forensicartifacts/artifacts/118)
<!-- Reviewable:end -->

darrenbilby commented:
Review status: 0 of 1 files reviewed at latest revision, 4 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 3 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K26-BbKOq2Wm7O_v0nY)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L3)):</sup>
Given you aren't making these should this be LinuxPythonModules?
<!-- Reviewable comment -K26-BbKOq2Wm7O_v0nZ:28303874 -->

---

<sup>**[definitions/modules.yaml, line 6 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25znv4_b_oRzK0IrCW)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L6)):</sup>
Why not in the doc string?
<!-- Reviewable comment -K25znv4_b_oRzK0IrCX:1726321638 -->

---

<sup>**[definitions/modules.yaml, line 18 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25zvgnfYllskcM7Gzn)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L18)):</sup>
If these are always one of a few values it may be more efficient as  {dist|site}
<!-- Reviewable comment -K25zvgnfYllskcM7Gzo:-1517911010 -->

---

<sup>**[definitions/modules.yaml, line 32 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25chaoaPOXhpQXDgoN)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L32)):</sup>
add a specifier for depth to the ** to make it explicit how deep to go e.g. **2
<!-- Reviewable comment -K25chaoaPOXhpQXDgoO:-1620916016 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,36 @@
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/.local/lib/python2.*/*-packages/*.egg-info'
```
What about /usr/shared/pyshared ?

joachimmetz replied:
Why only python 2.* ?

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,36 @@
+# Modules for interpreted languages.
+
+name: PythonModules
+doc: Python module files.
```
These look like user specific Python module files only ?

destijl replied:
The intent of this isn't obvious to me. Do you want just .py's? Do you just want information about eggs? Do you want .pyc's and .so's? Do you actually want to download everything, or just list the files?

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,36 @@
+    - '/usr/local/lib/python*/*-packages/*.egg-info'
+    - '/usr/local/lib/python*/*-packages/*.egg'
+  supported_os: [Linux]
+urls: ['https://pythonhosted.org/setuptools/formats.html']
```
What about the non-egg formats? e.g. wheel https://wheel.readthedocs.org/en/latest/

sebastianwelsh commented:
Review status: 0 of 1 files reviewed at latest revision, 7 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 3 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K26-BbKOq2Wm7O_v0nY)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L3)):</sup>
I've done the Linux targets. Might be worth adding the OSX/Windows equivalents in time. These can be added in with extra supported_os sources without needing a new artifact name though, right?
<!-- Reviewable comment -K2AhQNtowe3a9RASPXM:-1028271246 -->

---

<sup>**[definitions/modules.yaml, line 4 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r1-4)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L4)):</sup>
Clarified the type of data intended for collection, which is the packaging info files.

This isn't a firm guarantee of provenance or origin, but is an indicator that the modules are present.
<!-- Reviewable comment -K2Ah62miklJRiOwk277:1740935188 -->

---

<sup>**[definitions/modules.yaml, line 6 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25znv4_b_oRzK0IrCW)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L6)):</sup>
Done.
<!-- Reviewable comment -K2Ah2PsDj9TC2rSCV89:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 10 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r1-10)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L10)):</sup>
Added /usr/shared/pyshared
Removed 2.* match, which was a leftover from an earlier cut.
<!-- Reviewable comment -K2A1p34UuBrVPYrlxa4:591728997 -->

---

<sup>**[definitions/modules.yaml, line 18 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25zvgnfYllskcM7Gzn)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L18)):</sup>
Done.
<!-- Reviewable comment -K2AZEY-a-eA-MxYvWOa:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 23 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r1-23)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L23)):</sup>
Added. AFAICT they'll be discoverable through dist-info entries for the most part.
Also added cache entries to cover instances where they're installed into (say) virtualenv environments without needing to discover each virtualenv install.
<!-- Reviewable comment -K2AZFpkFo2ntDDNg6fo:127078172 -->

---

<sup>**[definitions/modules.yaml, line 32 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25chaoaPOXhpQXDgoN)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/modules.yaml#L32)):</sup>
AIUI the default is 3., but it seems like ruby applies more rigor than python. 2 fits the packaging format. Done.
<!-- Reviewable comment -K2AZcN2L7wxIkxMB7cI:-1141170382 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,104 @@
+    - '/usr/share/rubygems-integration/**2/*.gemspec'
+  supported_os: [Linux]
+urls: ['http://guides.rubygems.org']
+
```
- whiteline

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,104 @@
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/.local/lib/python*/(dist|site)-packages/*.egg-info'
```
Just FYI looking on my Fedora box I also see:
/usr/lib/python*
/usr/lib64/python*
/usr/local/lib64/python*/

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,104 @@
+    - '%%users.homedir%%/.cache/pip/wheels/*.whl'
+  supported_os: [Linux]
+labels: [Software]
+urls: ['https://wheel.readthedocs.org/en/latest/',
```
use
```
urls:
  - X
  - Y
```

joachimmetz commented on definitions/modules.yaml:
```diff
@@ -0,0 +1,104 @@
+    - '/usr/share/pyshared/*.egg-info'
+  supported_os: [Linux]
+labels: [Software]
+urls: ['https://pythonhosted.org/setuptools/formats.html',
```
use
```
urls:
- X
- Y
```

joachimmetz commented:
Some style nits and more common Python locations

destijl commented:
Review status: 0 of 1 files reviewed at latest revision, 13 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 4 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2DHYziUOPrs1FNXMxK)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L4)):</sup>
I don't think this syntax is correct, or it at least isn't consistent, see https://github.com/ForensicArtifacts/artifacts/blob/master/definitions/windows.yaml#L480
<!-- Reviewable comment -K2DHYziUOPrs1FNXMxL:-126422153 -->

---

<sup>**[definitions/modules.yaml, line 65 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2DHn3wP_NHBejQmKu0)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L65)):</sup>
change doc syntax as above.
<!-- Reviewable comment -K2DHn3wP_NHBejQmKu1:-1317729259 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

destijl commented:
Review status: 0 of 1 files reviewed at latest revision, 15 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 24 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2EelfiO0WpeuBZ3RYP)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L24)):</sup>
Artifacts aren't regex, they are globs, this should be:
`/usr/local/lib/python*/{dist,site}-packages/*.egg-info`
<!-- Reviewable comment -K2EelfiO0WpeuBZ3RYQ:-1233358802 -->

---

<sup>**[definitions/modules.yaml, line 29 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2EczIMZ2GzVxNds8cT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L29)):</sup>
I'm not sure what the purpose of DIRECTORY and FILE is here and in the others? On my machine `/usr/local/lib/python*/{dist,site}-packages/*.egg-info` are all directories. So you need to add extra *'s to get the files with FILE. Why do you want DIRECTORY as well? Can you please test this with GRR and come back with changes? I don't think it gets you what you want at the moment.
<!-- Reviewable comment -K2EczIMZ2GzVxNds8cU:1531123509 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

sebastianwelsh commented:
Added these locations, along with other changes that make the collection better.
<!-- Reviewable comment -K2KxSxm3m2rNhSV7Qiq:1923169816 -->

---

Reviewed 1 of 1 files at r2.
Review status: 0 of 1 files reviewed at latest revision, 15 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 4 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2DHYziUOPrs1FNXMxK)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L4)):</sup>
Done.
<!-- Reviewable comment -K2HKG6cy5t77B5i3vRc:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 20 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r2-20)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L20)):</sup>
Sure. The original intent was to collect modules installed from non-distro sources, but I see no disadvantage to adding these.
<!-- Reviewable comment -K2HSE6xdU3CJdWTDAOP:-1491553762 -->

---

<sup>**[definitions/modules.yaml, line 24 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2EelfiO0WpeuBZ3RYP)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L24)):</sup>
Done.
<!-- Reviewable comment -K2KrkjVPmzHo8VWGwcZ:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 29 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2EczIMZ2GzVxNds8cT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L29)):</sup>
.egg-infos make things messy, and I got this wrong. I've fixed this and removed the DIRECTORY. As you point out the right approach is to collect the files under the dir (when the target is actually a dir).

On your machine, all the files are dirs, but setuptools allows them to be either files or directories. I've changed the doc entry to make this clearer.
<!-- Reviewable comment -K2KslOBV9WSuugYHNQt:-1757038564 -->

---

<sup>**[definitions/modules.yaml, line 42 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r2-42)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L42)):</sup>
Done.
<!-- Reviewable comment -K2KtH5gziEDAnNzAOFv:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 65 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2DHn3wP_NHBejQmKu0)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L65)):</sup>
Done.
<!-- Reviewable comment -K2KtIieNVysy1jGBmq4:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 79 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r2-79)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L79)):</sup>
Done.
<!-- Reviewable comment -K2KxQYL240pNcjVRKg0:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 104 \[r2\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K25KAs5RZ7-12GKZ-XI-r2-104)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/8dbcaa81a3c806ad7af90efe34f7ee5125059635/definitions/modules.yaml#L104)):</sup>
Done.
<!-- Reviewable comment -K2Kx-cTtCgV7Fpx2ZJf:-499146009 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

destijl commented:
Review status: 0 of 1 files reviewed at latest revision, 14 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 1 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O6Mb8DqeIW_biqed8)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/modules.yaml#L1)):</sup>
It would be nice if we could come up with something more descriptive than modules.yaml, that could be almost anything. Maybe installed_src_code.yaml...? Open to better suggestions.
<!-- Reviewable comment -K2O6Mb8DqeIW_biqed9:-1074582166 -->

---

<sup>**[definitions/modules.yaml, line 24 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O5Wd-WwiTf3S8TS3B)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/modules.yaml#L24)):</sup>
Directories
<!-- Reviewable comment -K2O5Wd-WwiTf3S8TS3C:-1148930682 -->

---

<sup>**[definitions/modules.yaml, line 96 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O633a9QSQiuGha6gT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/modules.yaml#L96)):</sup>
Where you have one URL it should go on the same line, there's another above to fix as well. Joachim was referring to the trailing newline I think.
<!-- Reviewable comment -K2O633a9QSQiuGha6gU:-1568337939 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

sebastianwelsh commented:
Review status: 0 of 1 files reviewed at latest revision, 14 unresolved discussions.

---

<sup>**[definitions/modules.yaml, line 1 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O6Mb8DqeIW_biqed8)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/modules.yaml#L1)):</sup>
Some of the artifacts are source, others are already built. The common grouping is that they are installed_sw_modules, so renamed to that.
<!-- Reviewable comment -K2OowNEVQAhPF95YWbK:137375931 -->

---

<sup>**[definitions/modules.yaml, line 24 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O5Wd-WwiTf3S8TS3B)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/modules.yaml#L24)):</sup>
Done.
<!-- Reviewable comment -K2Opf_OSnR4ShIhrVQJ:-499146009 -->

---

<sup>**[definitions/modules.yaml, line 96 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O633a9QSQiuGha6gT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/modules.yaml#L96)):</sup>
Done.
<!-- Reviewable comment -K2OpiRoQ4koXzijAnnL:-499146009 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

destijl commented:
I think you probably forgot to push a couple of the other changes. But LGTM once the remaining minor changes are finished. We're also going to need a rebase to not pollute the git history with a ton of commits.
<!-- Reviewable comment -K2TBXvgquXOfU_7870i:-1626737577 -->

---

Reviewed 2 of 2 files at r4.
Review status: all files reviewed at latest revision, 13 unresolved discussions.

---

<sup>**[definitions/installed_src_sw.yanl, line 3 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K26-BbKOq2Wm7O_v0nY)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/installed_src_sw.yanl#L3)):</sup>
Lets add "(currently linux only)" or similar into the top line of the doc. I get that you want to avoid the name changing in the future since you're going to depend on it.
<!-- Reviewable comment -K2TAbaiKlm4w2HYl53v:-438294906 -->

---

<sup>**[definitions/installed_src_sw.yanl, line 24 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O5Wd-WwiTf3S8TS3B)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/installed_src_sw.yanl#L24)):</sup>
Looks like not done? :)
<!-- Reviewable comment -K2TBFUDSmj7_5UokaaL:147165079 -->

---

<sup>**[definitions/installed_src_sw.yanl, line 96 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O633a9QSQiuGha6gT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/installed_src_sw.yanl#L96)):</sup>
These ones aren't done either, you probably forgot a push.
<!-- Reviewable comment -K2TBSdI0b_Tkk6GF3Xi:1031026661 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

sebastianwelsh commented:
Yeah, my client ended up in a mess. I've rebased this cl and cleaned up superfluous commits.
<!-- Reviewable comment -K2eNtMEo-U8d27uFOpQ:-655581145 -->

---

Review status: 0 of 1 files reviewed at latest revision, 13 unresolved discussions.

---

<sup>**[definitions/installed_modules.yaml, line 3 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K26-BbKOq2Wm7O_v0nY)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/43832dc178d53d713373f2535d9600df3f2eab76/definitions/installed_modules.yaml#L3)):</sup>
Done.
<!-- Reviewable comment -K2eOrB3jUTkl3pUOVgq:-499146009 -->

---

<sup>**[definitions/installed_modules.yaml, line 24 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O5Wd-WwiTf3S8TS3B)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/installed_modules.yaml#L24)):</sup>
Done.
<!-- Reviewable comment -K2ePAVZFKzn2EYJf11U:-499146009 -->

---

<sup>**[definitions/installed_modules.yaml, line 96 \[r3\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118#-K2O633a9QSQiuGha6gT)** ([raw file](https://github.com/forensicartifacts/artifacts/blob/9bd3f01a16efe9551f52a902a810811e25c19adc/definitions/installed_modules.yaml#L96)):</sup>
Done.
<!-- Reviewable comment -K2ePCY0LjBZiDIs-K7n:-499146009 -->

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/118)
<!-- Sent from Reviewable.io -->

destijl commented:
LGTM
